back/backend/handlers/admin/edit_championships.py

```diff
@@ -0,0 +1,95 @@
+from flask import abort, Blueprint, redirect, render_template
+from google.cloud import ndb
+
+from backend.lib import auth, common
+from backend.models.championship import Championship
+from backend.models.region import Region
+from backend.models.province import Province
+from backend.models.user import Roles
+from backend.models.wca.competition import Competition
+from backend.models.wca.country import Country
+
+bp = Blueprint('edit_championships', __name__)
+client = ndb.Client()
+
+
+#@bp.route('/add_championship/<competition_id>/<championship_type>')
```

Comment:
Technically it would work without it but I think it's good to put the current advancement in common because it's the next step. Also it would be hard remove everything championship related from here since it would break other stuff.

If you want to save time you can just not review this file, since it's not going to be called until a new PR is made.

Reply:
I see. Please still make an Issue as per the other comment. (TODO comments in the code tend to get ignored more than Issues.)

Reply:
#55